WordPress/Classes/ViewRelated/Jetpack/Branding/JetpackBrandingVisibility.swift

```diff
@@ -1,30 +1,28 @@
-
-import Foundation
-
 /// An enum that unifies the checks to limit the visibility of the Jetpack branding elements (banners and badges)
+///
+/// This thing was born as an enum but at the time of writing had only one case in use.
+/// There is no reason to have that case linger around, it's been left merely to avoid changing its 15 usages.
```

Comment:
Admittedly, 15 usages are something I could tackle quickly enough. But I also didn't want to make this PR too noisy. Happy to followup if you feel this is important.

Also, I removed the unused cases under the assumption that, since they are unused, they are useless to us. I think this is a reasonable assumption as most of the Jetpack rebranding and migration work is done by now, right?

As a matter of fact, I wager we could further simplify this code. We shouldn't be too far from the point where we can more simply ask "are we running Jetpack, if so show|don't show element"